File: Basic/Conditions/219158W.c
#include <stdio.h>
#define intll long long

void solution() {

	int a, b, c;
	char op;

	scanf("%d %c %d = %d", &a, &op, &b, &c);

	if (op == '+') {
		if (a + b == c) {
			printf("Yes\n");
		} else {
			printf("%d\n", a + b);
		}
	} else if (op == '-') {
		if (a - b == c) {
			printf("Yes\n");
		} else {
			printf("%d\n", a - b);
		}
	} else {
		if (a * b == c) {
			printf("Yes\n");
		} else {
			printf("%d\n", a * b);
		}
	}
}

int main() {
	// char TEST_CASE = !1;
	int t = 1;
	// if (TEST_CASE == 1) {
	// 	scanf("%d", &t);
	// }

	while (t--) {
		solution();
	}
	return 0;
}

File: Basic/Conditions/219158W.go
package main

import (
	"fmt"
)

func solution() {

	var a, b, c int
	var op byte
	fmt.Scanf("%d %c %d = %d", &a, &op, &b, &c)

	if op == '+' {
		if a+b == c {
			fmt.Printf("Yes\n")
		} else {
			fmt.Printf("%d\n", a+b)
		}
	} else if op == '-' {
		if a-b == c {
			fmt.Printf("Yes\n")
		} else {
			fmt.Printf("%d\n", a-b)
		}
	} else {
		if a*b == c {
			fmt.Printf("Yes\n")
		} else {
			fmt.Printf("%d\n", a*b)
		}
	}
}

func main() {
	TEST_CASE := !true
	t := 1
	if TEST_CASE {
		fmt.Scanf("%d", &t)
	}

	for t != 0 {
		solution()
		t--
	}
}
